project/src/main/java/com/google/sps/Servlets/DocumentServlet.java

```diff
@@ -21,6 +21,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String hash = request.getParameter("documentHash");
     request.setAttribute("documentHash", hash);
 
-    request.getRequestDispatcher("/document.jsp#-" + hash).forward(request, response);
+    request.getRequestDispatcher("/document.jsp#" + hash).forward(request, response);
```

Comment:
was this the problem with not being able to load the correct doc?

Reply:
Yeah the hash includes the '-' so this was causing it to not find the hash